src/configuration.ts

```diff
@@ -866,6 +904,28 @@ export function getBuildLogForConfiguration(configuration: string | undefined):
     }
 }
 
+export function getPreConfigureArgsForConfiguration(configuration: string | undefined): void {
+    let makefileConfiguration: MakefileConfiguration | undefined = getMakefileConfiguration(configuration);
+    const preConfigArgs = makefileConfiguration?.preConfigureArgs;
+
+    if (preConfigArgs) {
+        configurationPreConfigureArgs = preConfigArgs;
+    } else {
+        configurationPreConfigureArgs = preConfigureArgs;
+    }
+}
+
+export function getPostConfigureArgsForConfiguration(configuration: string | undefined): void {
+    let makefileConfiguration: MakefileConfiguration | undefined = getMakefileConfiguration(configuration);
+    const postConfigArgs = makefileConfiguration?.postConfigureArgs;
+
+    if (postConfigArgs) {
```

Comment:
same postConfigArgs renaming localPostConfigureArgs, same reason as for Pre above.

Reply:
Done.